gc/base/MemoryManager.cpp

```diff
@@ -268,9 +301,9 @@ MM_MemoryManager::createVirtualMemoryForHeap(MM_EnvironmentBase *env, MM_MemoryH
 							}
 
 							/*
-							 * Attempt to allocate above 32G
+							 * Attempt to allocate heap at any location below ceiling including above 32G
 							 */
-							if ((NULL == instance) && (ceiling > THIRTY_TWO_GB_ADDRESS)) {
+							if (NULL == instance) {
```

Comment:
This is only line changed out of zLinux scope. This memory allocation attempt supposed to be last, like "try to allocate memory of requested size below ceiling at any location". Currently OpenJ9 set ceiling to 64G and making various attempts to allocate heap. This call in this case covers allocations of the heaps larger than 32G. So, existed code is correct for OpenJ9 purpose.
However in the case of standalone OMR usage the "last attempt" call might be missed if requested ceiling is under 32G and all previous attempts failed somehow.
If we want to keep scope of this PR zLinux only strictly I can revert this line change.

Reply:
I'm ok with keeping the change, since it's a simplification
however, I don't think the comment should refer to 32G anymore. should be generic like '...allocate heap at any location disregarding ceiling'